geopandas/tests/test_datasets.py

```diff
@@ -1,18 +1,13 @@
 from __future__ import absolute_import
 
+import pytest
+
 from geopandas import read_file, GeoDataFrame
 from geopandas.datasets import get_path
 
-
-class TestDatasets:
-
-    def test_read_paths(self):
-
-        gdf = read_file(get_path('naturalearth_lowres'))
-        assert isinstance(gdf, GeoDataFrame)
-
-        gdf = read_file(get_path('naturalearth_cities'))
-        assert isinstance(gdf, GeoDataFrame)
-
-        gdf = read_file(get_path('nybb'))
-        assert isinstance(gdf, GeoDataFrame)
+@pytest.mark.parametrize("test_gdf",
+                         [read_file(get_path('naturalearth_lowres')),
```

Comment:
Can you put the 'read_file(get_path' part inside the test? (so only parametrize the names of the datasets)

Reply:
Yup! Fixed.